src/OrchardCore.Modules/OrchardCore.Html/Drivers/HtmlBodyPartDisplayDriver.cs

```diff
@@ -43,7 +43,6 @@ public override IDisplayResult Display(HtmlBodyPart HtmlBodyPart, BuildPartDispl
             return Initialize<HtmlBodyPartViewModel>(GetDisplayShapeType(context), m => BuildViewModelAsync(m, HtmlBodyPart, context))
                 .Location("Detail", "Content:5")
                 .Location("Summary", "Content:10")
-                .Location("DetailAdmin", "Content:10") // For dashboard widgets;
```

Comment:
Intentional to remove these?

Why?

Reply:
Reusing existing `Detail` `DisplayType` and it's placement, so no need of  defining `DetailAdmin` Display type and it's placement.

Reply:
It can't be Detail. Detail is for front-end.

Reply:
UI content is same for `Detail` and `DetailAdmin`, why create new `DisplayType`?  Creating `DetailAdmin` display type means we need to define `Location` in drivers for every parts available in `OrchardCore` and duplicate the templates with same code

Reply:
Revert back to use `DetailAdmin`

Reply:
@dalenewman  @sebastienros  with `DetailAdmin` - display type

Currently adding a field or part (e.g. media field) in dashboard widget doesn't render them by default and requires to manually define placement for each field or part used

Reply:
I think we realized that with @agriffard and the solution is to add a default placement for DetailAdmin like we do for Detail.

Reply:
Apparently like the one you are removing here ;)